src/bots/baidu/ERNIEBot.js

```diff
@@ -1,10 +1,12 @@
 import Bot from "@/bots/Bot";
 import axios from "axios";
+// import i18n from "@/i18n";
 
 export default class ERNIEBot extends Bot {
   static _brandId = "ernie"; // ID of the bot, should be unique
   static _className = "ERNIEBot"; // Class name of the bot
-  static _logoFilename = "ernie-logo.png"; // Place it in public/bots/
+  static _logoFilename = "ernie-bot-logo.png"; // Place it in public/bots/
```

Comment:
```suggestion
  static _logoFilename = "ernie-logo.png"; // Place it in public/bots/
```

使用已经存在的 logo 是没问题的。我会改掉 API 版的 logo